pages/components/SignedUpComplete/SignedUpComplete.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+
+const SignedUpComponent = () => {
+  return(
+    <div className="flex items-center justify-center bg-none">
+      <div className="flex flex-col items-center justify-center bg-white rounded w-[600px] p-6 mt-[85px] text-black">
```

Comment:
For this line, remove "p-6 mt-[85px]" and add "px-6 py-10 mx-5".

The px adds padding horizontally
The py adds padding vertically
The mx adds a margin horizontally.